core/src/toga/widgets/webview.py

```diff
@@ -99,29 +103,34 @@ def user_agent(self, value):
     def set_content(self, root_url: str, content: str):
         """Set the HTML content of the WebView.
 
-        :param root_url: A URL which will be returned by the ``url`` property, and used
-            to resolve any relative URLs in the content. On Windows, this argument is
-            not currently supported, and calling this method will set the ``url``
-            property to ``None``.
+        **Note:** On Windows, the ``root_url`` argument is ignored. Calling this
+        method will set the ``url`` property to ``None``.
+
+        :param root_url: A URL which will be returned by the ``url`` property,
+            and used to resolve any relative URLs in the content.
         :param content: The HTML content for the WebView
         """
         self._impl.set_content(root_url, content)
 
     def evaluate_javascript(self, javascript, on_result=None) -> JavaScriptResult:
         """Evaluate a JavaScript expression.
 
-        **This method is asynchronous**. It does not guarantee that the provided
-        JavaScript has finished evaluating when the method returns. The object
-        returned by this method can be awaited to obtain the value of the expression,
-        or you can provide an ``on_result`` callback.
+        **Note:** This method is *asynchronous*. It does not guarantee that the
+        provided JavaScript has finished evaluating when the method returns. The
+        object returned by this method can be awaited to obtain the value of the
+        expression, or you can provide an ``on_result`` callback.
+
+        **Note:** On Android and Windows, *no exception handling is performed*.
+        If a JavaScript error occurs, a return value of None will be reported,
+        but no exception will be provided.
 
         :param javascript: The JavaScript expression to evaluate.
-        :param on_result: A callback that will be invoked when the JavaScript completes.
-            It should take one positional argument, which is the value of the
-            expression.
+        :param on_result: A callback that will be invoked when the JavaScript
+            completes. It should take one positional argument, which is the
+            value of the expression.
```

Comment:
It appears our editors are fighting with each other over the line length. Can we standardize on 88 columns for both docstrings and code, as we already have in Briefcase (https://github.com/beeware/briefcase/pull/1200)?